Gemfile.lock

```diff
@@ -217,4 +217,4 @@ DEPENDENCIES
   rubocop
 
 BUNDLED WITH
-   1.16.1
+   2.1.4
```

Comment:
This breaks current ChefDK, see also: https://github.com/chef-boneyard/chef-dk/issues/2149

Reply:
The README in the chefdk repo says:

`ChefDK is now EOL and will receive no further updates. It has been replaced with Chef Workstation. `

See here:

https://blog.chef.io/goodbye-chefdk-hello-chef-workstation

Can you try it with Chef Workstation? Or is maintaining compatibility with chefdk important?

Reply:
:+1: good point! Didn't see the depreciation note.